generator/internal/connection_generator.cc

```diff
@@ -64,7 +65,8 @@ Status ConnectionGenerator::GenerateHeader() {
        IsExperimental() ? "google/cloud/experimental_tag.h" : "",
        "google/cloud/version.h"});
   HeaderSystemIncludes(
-      {HasLongrunningMethod() ? "google/longrunning/operations.grpc.pb.h" : "",
+      {vars("proto_grpc_header_path"), vars("additional_pb_header_paths"),
```

Comment:
Consider `vars("proto_header_path")` instead of `vars("proto_grpc_header_path")`.  It seems we do not need the gRPC code, just the proto messages.